### ISSUE TITLE
Slave assigned to none

### ISSUE THREAD

**Issue by kar-dragon** [Free-Cities/Free-Cities#1029]
RA is assigning slaves to none, which not only is not a job, but is also removing slaves from facilities and important positions, like HG and recruiter

**Comment by prndev** [comment]
For me, this only happens with old saves. Resetting the assignment rules for every rule fixes behaviour. 
Does it happen with your new games, too?

**Comment by kar-dragon** [comment]
no, it only happened with my 10.6 save, but when i started NG+ with that save the rules start to work corretly again
